app/controllers/languages_controller.rb

```diff
@@ -5,6 +5,7 @@ def index
     @html_id = 'page'
     @body_id = 'languages'
     @locales = Locale.live
+    @title = PageTitle.new title_for(:languages)
```

Comment:
You don't have to use the `title_for()` helper here. You can construct whatever title text you want.

You could use the `I18n.locale` to look up the name/name in English/slug from Locale.

বাংলা / Bengali (baanlaa)

Reply:
Ohh okok, cool ! But since it's a page, we should use the page title mechanism, tho ?

Reply:
Creating a new `PageTitle` object is still a good idea. 👍🏻
But it's input arg can come from anywhere, not just `title_for()`.

Reply:
Good, but do you prefer the value to come from somehting like `headers > languages`? I'm worry to break the title if the menu title change or is just not use anymore as a string the the next design or whatever.

Perfect is the enemy better, so you can just tell me to not create a new set of key in the locale for now and wait to see for later.

Reply:
I think because the Locales table already has all of the information for this, it's better to use that than to add new entries to the locales YAML.